src/main/java/com/epam/ta/reportportal/entity/project/ProjectUtils.java

```diff
@@ -182,7 +182,7 @@ public static boolean doesHaveUser(Project project, String user) {
 	 */
 	@Nullable
 	public static ProjectUser findUserConfigByLogin(Project project, String user) {
-		return project.getUsers().stream().filter(it -> user.equals(it.getUser().getLogin())).findAny().orElse(null);
+		return project.getUsers().stream().filter(it -> user.equalsIgnoreCase(it.getUser().getLogin())).findAny().orElse(null);
```

Comment:
have you checked how login is stored in db after user created?

Reply:
All users' logins are stored in lower case